Resources/views/Translate/index.html.twig

```diff
@@ -12,7 +12,8 @@
             });
 
             $('.truncate-left').truncate({
-                side: 'left'
+                side: 'left',
+                width: 320
```

Comment:
This change actually makes it use less space.

Without `witdth: 320`:
![snimek z 2016-04-03 19-14-06](https://cloud.githubusercontent.com/assets/885946/14233712/87527b94-f9d0-11e5-92f5-2b856985e14e.png)

With `witdth: 320`:
![snimek z 2016-04-03 19-14-10](https://cloud.githubusercontent.com/assets/885946/14233711/7799176c-f9d0-11e5-815d-4ed30d8f1da5.png)